ios/browser/api/opentabs/brave_opentabs_api.h

```diff
@@ -0,0 +1,91 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_OPENTABS_BRAVE_OPENTABS_API_H_
+#define BRAVE_IOS_BROWSER_API_OPENTABS_BRAVE_OPENTABS_API_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, SyncDeviceType) {
+  SyncDeviceTypeUnset = 0,
+  SyncDeviceTypeWin,
+  SyncDeviceTypeMac,
+  SyncDeviceTypeLinux,
+  SyncDeviceTypeCros,
+  SyncDeviceTypeOther,
+  SyncDeviceTypePhone,
+  SyncDeviceTypeTablet
+};
```

Comment:
Not crucial to get this merged, but you can get away with no having to convert enums back and forth with their Chromium counterparts by using `NS_TYPED_ENUM` + consts, see: https://github.com/brave/brave-core/blob/master/ios/browser/api/url/url_formatter.h#L13-L19 and https://github.com/brave/brave-core/blob/master/ios/browser/api/url/url_formatter.mm#L24-L30

Reply:
Fixed. This actually helped find a bug where `SyncDeviceTypeTablet` was mapped to `default` enum case which was `UNSET` :)